app/notification_push.go

```diff
@@ -126,6 +126,22 @@ func (a *App) sendPushNotification(notification *postNotification, user *model.U
 	}
 }
 
+func (a *App) getFetchedPushNotificationMessage(postMessage, senderName, channelType string, hasFiles bool, userLocale i18n.TranslateFunc) string {
```

Comment:
what is the purpose of this function?

Reply:
yeah, you don't need this, cause this will try to build the message with the generic template, and we don't need that with this option